ScrabbleSinatra.rb

```diff
@@ -0,0 +1,49 @@
+require "sinatra"
+require "sinatra/reloader"
+require "./lib/Scrabble"
+
+class ScrabbleSinatra < Sinatra::Base
+  register Sinatra::Reloader
+
+  get "/" do
+    erb :home
+  end
+
+  get "/score" do
+    erb :score
+  end
+
+  post "/score" do
+    @word = params[:word]
+    # Using instance variable because refering to @word on score.erb.
+    # If local variable, score.erb is outside the scope.
+    @score = Scrabble::Scrabble.score(@word)
+    @letters = Scrabble::Scrabble.letter_array(@word)
+    @letter_scoring = Scrabble::Scrabble.letter_scoring(@letters)
```

Comment:
Nice work - it makes sense that all of these are instance variables since they are all utilized in your view.